pkgs/development/libraries/science/math/mkl/default.nix

```diff
@@ -156,6 +158,8 @@ in stdenvNoCC.mkDerivation {
   dontStrip = true;
   dontPatchELF = true;
 
+  passthru.tests = callPackage ./test { };
```

Comment:
This should ideally be attribute set containing tests.

Reply:
Ah, right, thanks! Fixed.